import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATIProcessor.java

```diff
@@ -67,22 +63,30 @@
     private List<Field> fieldList = new ArrayList<Field>();
     private List<Language> filterLanguages = new ArrayList<Language>();
 
+    private ActionStatus actionStatus;
+
 
     private boolean fromDatastore = false;
     public final static Set<String> IMPLEMENTED_VERSIONS = new HashSet<>
             (Arrays.asList("1.01", "1.03", "1.04", "1.05",
-            "2.01", "2.02","2.03"));
+                    "2.01", "2.02", "2.03"));
     //Variable that holds a MAP for tooltips and for fields translations
-    protected final static Map<String,Map<String,Map<String,Map<String,String>> >>langPack = new HashMap();
+    protected final static Map<String, Map<String, Map<String, Map<String, String>>>> langPack = new HashMap();
+
+    private static final String ISO_DATE = "yyyy-MM-dd";
+    private SimpleDateFormat df = new SimpleDateFormat(ISO_DATE);
+
 
     @Override
-    public  void setFromDataStore(boolean fromDatastore) {
+    public void setFromDataStore(boolean fromDatastore) {
```

Comment:
There are two setters for fromDatastore: setFromDataStore and setFromDatastore. Please remove one.

Reply:
done